plugin/pick.vim

```diff
@@ -56,7 +60,7 @@ function! s:FileListCommand()
 endfunction
 
 function! s:GitListCommand(file)
-  return "git ls-files " . a:file . " --cached --exclude-standard --others"
```

Comment:
As mentioned earlier, I agree that this change might be to opinionated
to include upstream. I ended up calling `PickCommand` directly with my
desired `git ls-files` command.